node/src/effect.rs

```diff
@@ -1662,6 +1663,20 @@ impl<REv> EffectBuilder<REv> {
             .await
     }
 
+    /// Announce that the sync process has finished.
+    pub(crate) async fn announce_finished_syncing(self)
```

Comment:
We have more than one notion of syncing in the node - I think we should be specific about this:
```suggestion
    pub(crate) async fn announce_finished_chain_syncing(self)
```

Reply:
Updated in 5b91f0f74616a4c60822d297c58f08523917ec38